R/getPath.R

```diff
@@ -16,8 +11,10 @@ getScenNamesFast <- function(outputDirs) {
 #' @rdname getPath
 #' @export
 getMifScenPath <- function(outputDirs, mustWork = FALSE) {
-  names <- getScenNamesFast(outputDirs)
-  path <- file.path(outputDirs, paste0("REMIND_generic_", names, ".mif"))
+  # Find all mif files starting with REMIND_generic_. The result will also contain REMIND_generic_withoutPlus.mif
+  reports <- list.files(outputDirs, "^REMIND_generic_.*\\.mif$", full.names = TRUE)
+  # Filter out filenames having "withoutPlus.mif" in their name
+  path <- reports[grepl("REMIND_generic_(?!.*(withoutPlus)\\.mif).*\\.mif$", reports, perl = TRUE)]
   normalizePath(path, mustWork = mustWork)
```

Comment:
```suggestion
  return(normalizePath(path, mustWork = mustWork))
```
I know you didn't change it, but I think an explicit return would make this much easier to read.